liota/dcc/dcc_base.py

```diff
@@ -53,4 +53,8 @@ def subscribe(self):
         pass
 
     def create_metric(self, gw, details, unit, sampling_function, sampling_interval_sec=10, aggregation_size=6):
+        self.publish_unit(gw, details, unit)
```

Comment:
should we make unit as an optional parameter, as not everything might have one (e.g., count of open file handles, etc) and passing null does not seem very intuitive? (Not actually related to your current changes.)